shared_python/Tags.py

```diff
@@ -103,6 +102,7 @@ def distinct_tags(self):
         original_description as "Original Description",
         '' as "TW Notes" FROM tags
       """)
+      #original_table, 'fanfiction_', '') as "Original Tag Type",                 #original_table not a column in eFiction code table
```

Comment:
I think we can remove this comment - the `original_table` column was for when the tags were being extracted directly from the original eFiction tables, but we don't care about the eFiction special code that used to hang out in this script anymore